src/cli/shell.rs

```diff
@@ -0,0 +1,219 @@
+use clap::Parser;
+use directories::UserDirs;
+use std::fs::{Permissions, exists, remove_file, set_permissions};
+use std::io::Write;
+use std::os::unix::fs::PermissionsExt;
+use std::path::PathBuf;
+use std::{
+    fmt::{self},
+    fs::{self, File},
+    path::Path,
+    str::FromStr,
+};
+
+use crate::cli::common::get_default_araki_bin_dir;
+
+#[derive(Parser, Debug)]
+pub struct Args {
+    #[clap(subcommand)]
+    subcommand: ShellSubcommand,
+}
+
+#[derive(Parser, Debug, Default)]
+pub struct ShellArg {
+    shell: Option<String>,
+}
+
+#[derive(Parser, Debug)]
+pub enum ShellSubcommand {
+    // Initialize the shell configuration (by editing ~/.bashrc etc)
```

Comment:
```suggestion
    /// Initialize the shell configuration (by editing ~/.bashrc etc)
```

Having the `///` instead of the `//` will make these comments show up in the cli help